docs/core/tutorials/upgrade.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Upgrade to a new .NET version
+description: Learn how to upgrade an app to a new .NET version.
+ms.topic: how-to
+ms.date: 03/25/2024
+---
+# Upgrade to a new .NET version
+
+New .NET versions are [released each year](https://github.com/dotnet/core/blob/main/releases.md). Many developers start the upgrade process as soon as the new version is available, while others wait until the version they are using is no longer supported. The upgrade process has multiple aspects to consider.
+
+## Requesting an upgrade (as a user)
+
+You may need an app to use a new .NET version and to make a rquest to the team or vendor that produced the app.
+
+Common reasons to upgrade to a new .NET version:
+
+- The currently used .NET version is no longer supported
+- The new version supports a new operating system
+- The new version has an important performance or security feature
+
+## Upgrading development environment
+
+The .NET SDK is the primary component to install, to upgrade to a new .NET version. It includes an updated .NET CLI, build system, and runtime version. 
+
+The .NET website offers [installers and archives](https://dotnet.microsoft.com/download) that can be downloaded and used on any supported operating system and architecture.
+
+Some operating systems have a package manager that can be used to install a new .NET version, which may be a prefered option for some users.
+
+- [macOS](https://formulae.brew.sh/cask/dotnet-sdk)
+- [Linux](https://learn.microsoft.com/dotnet/core/install/linux)
+- [Windows](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/DotNet/SDK)
+
+Visual Studio installs [new .NET SDK versions automatically](https://learn.microsoft.com/dotnet/core/porting/versioning-sdk-msbuild-vs). For Visual Studio users, upgrading to a newer Visual Studio version will be sufficient.
+
+## Upgrading source code
+
+The only required change to upgrade an app is updating the `TargetFramework` property in a project file to the newer .NET version.
+
+Here's how to do it:

```

Comment:
We have tooling for upgrades, we should absolutely be mentioning them https://aka.ms/dotnet-upgrade-assistant -- it helps automate all of this for you and give you a nice report of more tasks you might need to do!

Reply:
You should probably link here instead: https://learn.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-overview (../porting/upgrade-assistant-overview.md)